README.md

```diff
@@ -17,43 +17,40 @@ in the scripts folder and understand as much as possible. We'll touch some
 things briefly here that are better explained there. The steps here will expect
 you have some knowledge of what's going on inside the script.
 
-  1. Checkout LLVM to a to your home directory from
-     [OUR FORK](https://github.com/cmput415/llvm-project), checkout the 10.0.0
-     release, and change the directory in your script.
+  1. Checkout LLVM to your machine
      1. `cd $HOME`
-     1. `git clone git@github.com:cmput415/llvm-project.git`
-     1. `cd llvm`
-     1. `git checkout llvmorg-10.0.0`
-  1. Add these configuration lines to your `~/.bashrc` on linux or
-     `~/.bash_profile` on MacOS to setup the next steps. You should restart your
+     1. `git clone https://github.com/llvm/llvm-project.git`
+     1. `cd llvm-project`
+     1. `git checkout llvmorg-16.0.4`
+  1. Build MLIR
+     1. `mkdir build`
+     1. `cd build`
+     1. ```cmake -G Ninja ../llvm \
+             -DLLVM_ENABLE_PROJECTS=mlir \
+             -DLLVM_BUILD_EXAMPLES=ON \
```

Comment:
Are you sure you really want to build examples? I've found it to be a waste of build time in a lot of cases. Does it do something interesting for MLIR?

Reply:
The important example for MLIR is called "Toy". Unfortunately, they build a new dialect, which is way beyond what we want to do, but they do have lots of good MLIR generation examples. The big thing about dialects is that you specify them in template files that are then parsed and used to generate classes and methods. You only get this generated code if you build the example, and there's lot of good stuff in the generated files. This is also why "git grep" isn't as good as "grep -R".